api/src/main/java/fr/catcore/server/translations/api/mixin/MixinTranslatableText.java

```diff
@@ -30,7 +34,9 @@ public Text asLocalizedFor(LocalizationTarget target) {
             this.updateTranslations();
 
             MutableText literal = this.selfAsLiteral(target);
-
+            if (literal instanceof LiteralText && this.key.equals(literal.getString())) {
```

Comment:
hmm, what does this do?

Reply:
if the literal LiteralText is equals to the TranslatableText key, it sends the key as a translatable text.
else it would have been sent to the client as a LiteralText and then not translated to the client language.

It should only apply to vanilla keys.

Reply:
Ah! I understand. It might be worth having this logic take place in `selfAsLiteral` itself? Or even `LocalizableText.asLocalizedFor` might make most sense

Reply:
moved into `selfAsLiteral`